src/pages/en/community/guidelines.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Guidelines
+description: Quilt's community guidelines
+layout: /src/layouts/Page.astro
+---
+
+import Message from "@atoms/Message.astro";
+
+# What is this page?
+
+This page is a living document that outlines the community guidelines for Quilt commmunity. While those guidelines aren't authoritative,
+they set a standard for what we expect from our community members and staff.
+
+Authoritative policies can be found in our [RFCs](/about/faq#whats-an-rfc-how-do-they-work), along with our [Code of Conduct](/community/code-of-conduct) and [rules](/community/rules).
+If a guideline is in conflict with one of these, that document takes precedence over the guidelines.
+
+# Moderation
+
+<Message>
+	# Reporting a staff member
+
+	If you feel the need to report a member of staff, the right person to contact will depend on their roles:
+	- Developer: Contact ModMail or a Moderator directly
+	- Moderator: Contact a Manager or ModMail
+	- Manager: Contact a member of the Administrative Board or another Manager
+	- Administrative Board: Contact a Manager or another member of the Administrative Board
+</Message>
+
+<Message>
+	# Who can see a ModMail thread?
+
+	ModMail threads are, in most cases, only seen by Moderators and Community Managers. However, we may choose to share a thread with the Administrative Board on request,
+	for oversight purposes.
+
+	Application threads will also be seen by the Administrative Board and Community Team, as they are a part of the application process.
+
+	We may also choose to share a stripped down version of the thread with the Community Team, with any personal information redacted, if required as part of an
+	action against a member of the staff.
+
+	If we would like to share a thread further, we will ask for your permission first. Please note that bad faith threads are exempt from this rule.
```

Comment:
we have also once pulled the CT into an ban appeal thread as it concerned some of the members, would that be prohibited in this case, or require stripping it down &| asking permission?